docs/source/tutorials/running_a_simulation/interactive.rst

```diff
@@ -0,0 +1,631 @@
+.. _interactive_tutorial:
+
+==================================
+Running a Simulation Interactively
+==================================
+
+In this tutorial, we'll walk through getting a simulation up and running in an
+interactive setting such as a Python interpreter or Jupyter notebook and
+do a little exploration of the simulation model.
+
+Running a simulation in this way is useful for a variety of reasons, foremost
+for debugging and validation work. It allows for changing simulation
+:term:`configuration <Configuration>` programmatically, stepping through a
+simulation in a controlled fashion, and examining the
+:term:`state <State Table>` of the simulation itself.
+
+
+For the following tutorial, we will assume you have set up an environment and
+installed ``vivarium``. If you have not, please see the
+:ref:`Getting Started <getting_started_tutorial>` section.  We'll be using
+the :ref:`disease model <disease_model_tutorial>` constructed
+in a separate tutorial for exploration purposes here. The
+:ref:`components <components_concept>` constructed in that tutorial are
+available in the ``vivarium`` package, so you don't need to build them yourself
+before starting this tutorial.
+
+.. contents::
+   :depth: 2
+   :local:
+   :backlinks: none
+
+What are we making?
+-------------------
+
+Simulations are complicated things. It's beyond the scope of this tutorial
+in particular to talk about what they are and how they work and when they
+make sense as models of the world. Luckily, once you have one in hand, you
+can start figuring out the answers to many of those questions yourself.
+
+In this tutorial, we're going to put together and examine an individual-based
+epidemiology model from a bunch of pre-constructed parts. We'll start out
+rather mechanically, just showing how to set up and run a simulation and pull
+out interesting data. As we go on, we'll talk about what we sort of results
+we should expect from the structure of the model and how we can verify those
+expectations.
+
+Our model starts with a bunch of people with uniformly distributed ages and
```

Comment:
I'm not sure this paragraph belongs here. This tutorial doesn't really touch on anything about what the model is doing - just how to run a model.

Reply:
That's how I perceive this tutorial as well. I see the exposition of the model as laying in the disease model section with a ref at the top, whereas this tutorial is all about the detail of running the thing, not about what the model actually is